src/ai/backend/agent/agent.py

```diff
@@ -858,6 +860,16 @@ async def _pipeline(r: Redis):
                 )
             )
 
+        _container_observer_task = HeartbeatTask(
+            self.id,
+            self.enumerate_containers,
+            self.event_producer,
+        )
+        self._container_observer = TaskRunner(
+            TaskRunnerArgs(task=_container_observer_task, interval=10, continue_on_error=True)
+        )
```

Comment:
The interval seems too short. Also, magic numbers like DEFAULT values should be defined as separate `FINAL` values.